lib/scala/pkg/src/main/scala/org/enso/pkg/Package.scala

```diff
@@ -9,7 +9,18 @@ import java.net.URI
 import scala.jdk.CollectionConverters._
 import scala.util.{Failure, Try, Using}
 
-object CouldNotCreateDirectory extends Exception
+case class CouldNotCreateDirectory(cause: Throwable) extends RuntimeException {
+  override def getMessage: String =
+    s"Could not create directory: ${cause.getMessage}. Perhaps there is a permission issue."
```

Comment:
I added the comment, as for me this was caused by https://learn.microsoft.com/en-us/defender-endpoint/controlled-folders#what-is-controlled-folder-access